home-api-tests/pom.xml

```diff
@@ -310,5 +316,16 @@
             <artifactId>commons-lang3</artifactId>
             <version>${commons-lang3-version}</version>
         </dependency>
+        <dependency>
+            <groupId>io.github.perplexhub</groupId>
+            <artifactId>rsql-common</artifactId>
```

Comment:
Can we get rid of this dependency in tests?

Reply:
done